jenkins/ec2_bootstrap.sh

```diff
@@ -423,6 +423,24 @@ prepare_cowbuilder() {
     /bin/bash ./bootstrap.sh
 }
 
+install_datadog_agent() {
+    # Install the Datadog agent. The agent requires an API key which is expected
+    # to be present in the shell environment as DATADOG_AGENT_API_KEY. If the
+    # value is not present then inform stdout but do not sys exit.
+    if ! [[ -z "${DATADOG_AGENT_API_KEY}" ]]; then
+	echo "Optional envvar DATADOG_AGENT_API_KEY does not exist."
+    else
+	DD_CURL_TO_BASH_URI="https://raw.githubusercontent.com/DataDog/dd-agent/master/packaging/datadog-agent/source/install_agent.sh"
+	DD_AGENT_CONFIG_FILE='/etc/dd-agent/datadog.conf'
+	DD_AGENT_DAEMON='datadog-agent'
+	
+	DD_API_KEY="${DATADOG_AGENT_API_KEY}" \
+		  bash -c "$(curl -L ${DD_CURL_TO_BASH_URI})" && \
```

Comment:
There's no need to save the script's contents in a variable. Here's how to stream it:

curl -L $DD_CURL_TO_BASH_URI | bash -